src/main/kotlin/graphics/scenery/DFTParser.kt

```diff
@@ -0,0 +1,132 @@
+package graphics.scenery
+import graphics.scenery.repl.REPL
+import graphics.scenery.utils.RingBuffer
+import org.joml.Vector3f
+import org.lwjgl.system.MemoryUtil
+import java.io.File
+import java.nio.ByteBuffer
+
+/**
+Parses a density functional theory (common simulation method in solid state physics and theoretical chemistry)
+calculation. Can be used to visualize single DFT calculations or DFT-MD (=DFT molecular dynamics simulations).
+
+ * @property[normalizeDensityTo] Defines to which value the density is scaled. This is useful when visualizing more then one DFT calculation at the
+    same time, in order to keep the density visualization consistent. Negative values mean the density is scaled
+    per snapshot. Default is -1.0f.
+ */
+class DFTParser (val normalizeDensityTo: Float = -1.0f){
+    /**  Number of Atoms.*/
+    var numberOfAtoms:Int = 0
+    /** Distance between two grid points in either direction in Bohr.*/
+    var gridSpacings = FloatArray(3) { 0.0f }
+    /** Number of gridpoints in 3D grid.*/
+    var gridDimensions = IntArray(3) { 0 }
+    /** Positions of the atoms in Bohr.*/
+    var atomicPositions = Array(0){ Vector3f()}
+    private var electronicDensity = Array(0, {Array(0, { FloatArray(0) } ) } )
+    /** Electronic density as scaled bytes, in scaled(1/Bohr).*/
+    lateinit var electronicDensityUInt : ByteBuffer
```

Comment:
Minor thing, but this is public now, you might wanna declare it as 
```kotlin
lateinit var electronicDensityUInt: ByteBuffer
    protected set
```

... which means you can get it public, but not reassign it 👍

Reply:
Ah yes, good idea, done!